<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RxChannel</class>
 <widget class="QWidget" name="RxChannel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>79</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="grpChannelName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Canal 1</string>
   </property>
   <widget class="QCheckBox" name="chkChannelActive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>78</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ativo</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChannelColor">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cor</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblColor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
